ghost/core/package.json

```diff
@@ -231,11 +231,13 @@
     "eslint": "8.37.0",
     "expect": "29.3.1",
     "form-data": "4.0.0",
+    "ghost-admin": "5.46.0",
```

Comment:
Do you reckon we can use a `*` here? It's gonna be a pain to keep the version in sync (_more_ code in Ghost-Release) and I'd like to clean up the process that means we need this, but not necessarily now